hazelcast/src/main/java/com/hazelcast/map/impl/operation/MapOperation.java

```diff
@@ -55,7 +55,8 @@
      * Used by wan-replication-service to disable wan-replication event publishing
      * otherwise in active-active scenarios infinite loop of event forwarding can be seen.
      */
-    protected boolean disableWanReplicationEvent;
+    protected transient boolean disableWanReplicationEvent;
```

Comment:
Opposed to `canPublishWanEvent`, `disableWanReplicationEvent` is serialized in some operations. I think it should be like this:
```
protected boolean disableWanReplicationEvent;
private transient boolean canPublishWanEvent;
```

Reply:
Oops, should be ok now.